src/wrappers/themis/ruby/lib/rbthemis.rb

```diff
@@ -274,7 +274,7 @@ def initialize(id, private_key, transport)
       @session = secure_session_create(
         id_buf, id_length, private_key_buf, private_key_length, @callbacks)
 
-      raise ThemisError, 'Secure Session failed creating' unless @session
+      raise ThemisError, 'Secure Session failed creating' unless !@session.null?
```

Comment:
You can use `if` here instead of  `unless` to avoid extra negation:

```ruby
raise ThemisError, 'Secure Session failed creating' if @session.null?
```

which makes the code a bit more readable.